src/Service/Services/ExecutionHelper.cs

```diff
@@ -208,24 +227,33 @@ private static object PreParseLeaf(IMiddlewareContext context, string leafJson)
 
         public static bool RepresentsNullValue(JsonElement element)
         {
+            // TODO: why does this not check the element kind?
             return string.IsNullOrEmpty(element.ToString()) && element.GetRawText() == "null";
```

Comment:
Why is this done this way? We could just use the kind since this allocates.

Reply:
Yes, good point.